driver/modern_bpf/helpers/interfaces/attached_programs.h

```diff
@@ -13,7 +13,10 @@
 /* This enum is used to tell if we are considering a syscall or a tracepoint */
 enum intrumentation_type
 {
-	SYSCALL = 0,
+	//SYSCALL definition is skipped on powerpc since SYSCALL is already defined in vmlinux.h
+	#ifndef __TARGET_ARCH_powerpc
+		SYSCALL = 0,
```

Comment:
I think we just need to avoid name collision, by prefixing these ones with eg: `MODERN_BPF_`.
@Andreagit97 thoughts?

Reply:
@mdafsanhossain here !

Reply:
@FedeDP Do you mean something like this? `MODERN_BPF_SYSCALL=0`? I am not sure if I am understanding this correctly.

Reply:
```
enum intrumentation_type
{
               MODERN_BPF_SYSCALL = 0,
               MODERN_BPF_TRACEPOINT = 1,
}
```
and then rename their usages whereveer `sampling_logic` is called.

Reply:
yes I agree, probably we can rename them in `INSTR_TYPE_SYSCALL` and `INSTR_TYPE_TRACEPOINT`

Reply:
uhm I didn't see new messages, if we all agree it's also okay to go with 
```c
enum intrumentation_type
{
               MODERN_BPF_SYSCALL = 0,
               MODERN_BPF_TRACEPOINT = 1,
}
```

Reply:
Makes sense now. Thanks :)